Documentation/AdvancedUsage.md

```diff
@@ -951,5 +951,3 @@ There are some important things to remember when using network reachability to d
     - Even though the network requests may still fail, this is a good moment to retry them.
 - The network reachability status can be useful for determining why a network request may have failed.
     - If a network request fails, it is more useful to tell the user that the network request failed due to being offline rather than a more technical error, such as "request timed out."
-
```

Comment:
Perhaps there's a modern talk along similar lines?

Reply:
That's a great question @jshier. I'm honestly not sure at all. Might take a really long time to track down that information. I think for now we remove this and add another link back in if we track one down.